lib/temperature.js

```diff
@@ -206,6 +206,34 @@ var Controllers = {
       }
     }
   },
+
+  TMP102: {
+    initialize: {
+      value: function(opts, dataHandler) {
+        this.io.i2cConfig(opts);
+
+        this.io.i2cRead(0x48, 0x00, 2, function(data) {
```

Comment:
It is addressable but the docs are pretty abysmal. It's unclear what addresses are available. The TMP102 itself has 4 addresses but the breakout board from Sparkfun only seems to have two. :confused: